docs/tutorials/pre_executed/nestedframe.ipynb

```diff
@@ -0,0 +1,2041 @@
+{
```

Comment:
<p>That's some funky formatting in the code cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/838/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>